layers/+completion/ivy/packages.el

```diff
@@ -184,7 +183,8 @@
       (ido-mode -1))))
 
 (defun ivy/init-ivy-hydra ()
-  (use-package ivy-hydra))
+  (use-package ivy-hydra)
+  (define-key hydra-ivy/keymap [escape] 'hydra-ivy/keyboard-escape-quit-and-exit))
```

Comment:
Actually nevermind. I tested it myself and realised that we need to require `ivy-hydra` here, otherwise I get an error when pressing `C-o` in an Ivy prompt:
> Symbol’s function definition is void: ivy--matcher-desc